libs/tools/barista/src/main.ts

```diff
@@ -109,6 +109,11 @@ async function buildPages(): Promise<void[]> {
       const path = result.relativeOutFile
         .replace(/^\//, '') // replace the leading slash
         .replace(/\..+$/, ''); // replace the file ending
+
+      if (path === 'index') {
```

Comment:
Isn't it `home` instead of `index` now?

Reply:
nope it is index again :) Checkout the commit history there is the explanaition